FacilityDB/3 reports.sql

```diff
@@ -0,0 +1,31 @@
+--AS: you don't have proper sample code for most of your questions, just letting you know.
+--The total amount of Telehealth appts. comparing the last three years.
+select TotalTelehealth = count(*), Year = year(a.AppointmentDateTime)
+from Appointments a
+group by year(a.AppointmentDateTime)
+having year(a.AppointmentDateTime) in (2021,2022,2023)
+--The number of days prior to the appts. that the appts. are scheduled, comparing the Telehealth appts. to the in-person appts.
+--AS I am assuming that the appointments were scheduled today, since you don't have a column for day scheduled. 
```

Comment:
Your assumption is fine but I really wanted a column that records the date of when the appt. is scheduled. (updated my sample data)

Reply:
I added in my own sample data since I didn't see this till after I already added in the new column you asked for and the sample data.

Reply:
done